backend/src/models/v2/File.ts

```diff
@@ -44,6 +45,8 @@ const FileSchema = new Schema<FileInterface>(
   },
 )
 
+FileSchema.plugin(MongooseDelete, { overrideMethods: 'all', deletedBy: true, deletedByType: Schema.Types.ObjectId })
```

Comment:
What does the plugin() do?

Reply:
This allows us to "delete" documentns from a Mongo collection. Normally, you can just delete a document but that means you can't undelete it. This plugin allows us to mark documents as "deleted", which means they won't be retrieved via normal GET requests, but they still exist in the collection so that we can make special requests to restore them.